### ISSUE TITLE
The tab overflow ("three dots") button doesn't show on certain window sizes

### ISSUE THREAD

**Issue by Dartvauder** [gradio-app/gradio#9682]
### Describe the bug

Hi. Before the update to version 5.0.2 from 4.44.1, if the tabs did not fit into one line, a second one was created. But when I updated, the entire second line disappeared and everything tries to fit into one line, which does not happen.

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```
with gr.TabbedInterface(
    [
        gr.TabbedInterface(
            [chat_interface, tts_stt_interface, mms_interface, seamless_m4tv2_interface, translate_interface],
            tab_names=[_("LLM", lang), _("TTS-STT", lang), _("MMS", lang), _("SeamlessM4Tv2", lang), _("LibreTranslate", lang)]
        ),
        gr.TabbedInterface(
            [
                gr.TabbedInterface(
                    [txt2img_interface, img2img_interface, depth2img_interface, marigold_interface, pix2pix_interface, controlnet_interface, latent_upscale_interface, supir_upscale_interface, sdxl_refiner_interface, inpaint_interface, outpaint_interface, gligen_interface, diffedit_interface, blip_diffusion_interface, animatediff_interface, hotshotxl_interface, video_interface, ldm3d_interface,
                     gr.TabbedInterface([sd3_txt2img_interface, sd3_img2img_interface, sd3_controlnet_interface, sd3_inpaint_interface],
                                        tab_names=[_("txt2img", lang), _("img2img", lang), _("controlnet", lang), _("inpaint", lang)]),
                     cascade_interface, t2i_ip_adapter_interface, ip_adapter_faceid_interface, riffusion_interface],
                    tab_names=[_("txt2img", lang), _("img2img", lang), _("depth2img", lang), _("marigold", lang), _("pix2pix", lang), _("controlnet", lang), _("upscale(latent)", lang), _("upscale(SUPIR)", lang), _("refiner", lang), _("inpaint", lang), _("outpaint", lang), _("gligen", lang), _("diffedit", lang), _("blip-diffusion", lang), _("animatediff", lang), _("hotshotxl", lang), _("video", lang), _("ldm3d", lang), _("sd3", lang), _("cascade", lang), _("t2i-ip-adapter", lang), _("ip-adapter-faceid", lang), _("riffusion", lang)]
                ),
                kandinsky_interface, flux_interface, hunyuandit_interface, lumina_interface, kolors_interface, auraflow_interface, wurstchen_interface, deepfloyd_if_interface, pixart_interface, playgroundv2_interface
            ],
            tab_names=[_("StableDiffusion", lang), _("Kandinsky", lang), _("Flux", lang), _("HunyuanDiT", lang), _("Lumina-T2X", lang), _("Kolors", lang), _("AuraFlow", lang), _("Würstchen", lang), _("DeepFloydIF", lang), _("PixArt", lang), _("PlaygroundV2.5", lang)]
        ),
        gr.TabbedInterface(
            [wav2lip_interface, liveportrait_interface, modelscope_interface, zeroscope2_interface, cogvideox_interface, latte_interface],
            tab_names=[_("Wav2Lip", lang), _("LivePortrait", lang), _("ModelScope", lang), _("ZeroScope2", lang), _("CogVideoX", lang), _("Latte", lang)]
        ),
        gr.TabbedInterface(
            [stablefast3d_interface, shap_e_interface, sv34d_interface, zero123plus_interface],
            tab_names=[_("StableFast3D", lang), _("Shap-E", lang), _("SV34D", lang), _("Zero123Plus", lang)]
        ),
        gr.TabbedInterface(
            [stableaudio_interface, audiocraft_interface, audioldm2_interface, bark_interface, rvc_interface, uvr_interface, demucs_interface],
            tab_names=[_("StableAudio", lang), _("AudioCraft", lang), _("AudioLDM2", lang), _("SunoBark", lang), _("RVC", lang), _("UVR", lang), _("Demucs", lang)]
        ),
        extras_interface,
        gr.TabbedInterface(
            [wiki_interface, gallery_interface, model_downloader_interface, settings_interface, system_interface],
            tab_names=[_("Wiki", lang), _("Gallery", lang), _("ModelDownloader", lang), _("Settings", lang), _("System", lang)]
        )
    ],
    tab_names=[_("Text", lang), _("Image", lang), _("Video", lang), _("3D", lang), _("Audio", lang), _("Extras", lang), _("Interface", lang)],
    theme=theme
) as app:
```

### Screenshot

In 4.44.1 version:

<img width="1112" alt="2" src="https://github.com/user-attachments/assets/ec0837c6-4ef5-499f-885d-8bca79bfbf9f">

In 5.0.2 version:

<img width="1109" alt="1" src="https://github.com/user-attachments/assets/9609f5e7-0c5a-4899-807d-eb9a7f575336">

### Logs

If I click on the very edge of the tab list, I get to the tab that is not visible. Maybe in this case, add scrolling via the arrows on the sides?

### System Info

```shell
Gradio Environment Information:
------------------------------
Operating System: Windows
gradio version: 5.0.2
gradio_client version: 1.4.0

------------------------------------------------
gradio dependencies in your environment:

aiofiles: 23.2.1
anyio: 4.3.0
fastapi: 0.110.2
ffmpy: 0.3.2
gradio-client: 1.4.0
httpx: 0.27.0
huggingface-hub: 0.25.2
jinja2: 3.1.3
markupsafe: 2.1.5
numpy: 1.26.4
orjson: 3.10.1
packaging: 24.1
pandas: 1.5.3
pillow: 10.3.0
pydantic: 2.7.1
pydub: 0.25.1
python-multipart: 0.0.9
pyyaml: 6.0.1
ruff: 0.4.2
semantic-version: 2.10.0
tomlkit: 0.12.0
typer: 0.12.3
typing-extensions: 4.11.0
urllib3: 2.2.1
uvicorn: 0.29.0
authlib; extra == 'oauth' is not installed.
itsdangerous; extra == 'oauth' is not installed.


gradio_client dependencies in your environment:

fsspec: 2024.3.1
httpx: 0.27.0
huggingface-hub: 0.25.2
packaging: 24.1
typing-extensions: 4.11.0
websockets: 12.0
```

### Severity

I can work around it

**Comment by Dartvauder** [comment]
I found a short-term solution to the problem. When I delete the URL link and go back to the interface, magically this appears:

![image](https://github.com/user-attachments/assets/873115db-d735-42fa-9f6c-b859127346b9)

and

![image](https://github.com/user-attachments/assets/54f166ca-fb01-4bfc-9e10-b7fc78b09fe7)

**Comment by pngwn** [comment]
Hmm. This menu should always be visible if there are too many tabs to fit on the page.

@hannahblair Do you have any idea what could be happening here?

**Comment by abidlabs** [comment]
@Dartvauder could you provide a self-contained repro that we can run to reproduce the issue? I'm not sure I follow what's happening here.

**Comment by Salonen-github** [comment]
I have this same problem. When tabbedinterface is used and if the names of the menu in the top bar cannot all be displayed on the line, then the three dots with which the rest of the menu items can be displayed will not appear immediately. Those three dots will only appear if you adjust the window size. As a clarification, the very top menu works as it should and the three dots are always visible. But the problem occurs in the submenus. Appears on both Chrome and Edge. I haven't tested other browsers.

**Comment by rsxdalv** [comment]
Is there any way to get back the old behavior? The new UI basically 'punishes' having too many tabs since they are very inconvenient to switch between (user story - an user wants to switch between the last 2-3 tabs back and forth but they are 'off the page').
I get that it looks ugly but I prefer gradio to be robust not pretty.

**Comment by abidlabs** [comment]
Hi folks, we'll definitely get the the three dots to show on all window sizes. But @rsxdalv can you open a new issue for that particular feature request? We'll consider seeing if we can support it in a way that is consistent with the Gradio 5 design

**Comment by abidlabs** [comment]
Actually I can't either @hannahblair. What browser are y'all using @Dartvauder @Salonen-github?

**Comment by Dartvauder** [comment]
> Actually I can't either @hannahblair. What browser are y'all using @Dartvauder @Salonen-github?

I am using Opera, Chrome, DuckDuckGo and Firefox. In all of this browsers i have this problem

**Comment by Salonen-github** [comment]
> Actually I can't either @hannahblair. What browser are y'all using @Dartvauder @Salonen-github?

Chrome and Edge. Both are updated to latest version.
Gradio version is 5.1.0, but will be updated when the latest version comes available via pip.

**Comment by Dartvauder** [comment]
I confirm. For me, this problem was also solved with the update to 5.3.0

We can consider the problem solved. Thank you very much
